src/app.ts

```
@@ -49,12 +55,19 @@ export default async function newApp(): Promise<Express> {
   return app;
 }
 
-function asyncHandler(func: RequestHandler): RequestHandler {
+type AsyncRequestHandler = (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => Promise<void>;
+
+function asyncHandler(func: AsyncRequestHandler): AsyncRequestHandler {
   return (req: Request, res: Response, next: NextFunction) =>
-    Promise.resolve(func(req, res, next)).catch(next);
+    func(req, res, next).catch(next);
```

this will still silently ignore any error